### PR TITLE
feat(ribbon): update warp tokens

### PR DESCRIPTION
**Pull request by CarolineGard**
Update ribbon with warp tokens

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,15 +25,16 @@ export const pageIndicator = {
...
 export const ribbon = {
-  base: 'py-4 px-8 border rounded-4 inline-flex last-child:mb-0',
-  info: 'border-aqua-50 bg-aqua-50',
-  success: 'border-green-100 bg-green-100',
-  warning: 'border-yellow-100 bg-yellow-100',
-  error: 'border-red-100 bg-red-100',
-  disabled: 'border-bluegray-300 bg-bluegray-300',
-  sponsored: 'border-aqua-200 bg-aqua-200',
-  neutral: 'border-gray-100 bg-gray-100',
-  outlined: 'border-bluegray-300 bg-white',
```
Comment (AnnaRybkina) [inline index.js:36]: Is `outlined` removed on purpose?

File: index.js [hunk 2]
```diff
@@ -25,15 +25,16 @@ export const pageIndicator = {
...
+  base: 'py-4 px-8 border rounded-4 inline-flex last:mb-0',
+  info: ' i-bg-$color-badge-info-background i-text-$color-badge-info-text',
+  success: ' i-bg-$color-badge-positive-background i-text-$color-badge-positive-text',
+  warning: ' i-bg-$color-badge-warning-background i-text-$color-badge-warning-text',
+  error: ' i-bg-$color-badge-negative-background i-text-$color-badge-negative-text',
+  disabled: 'i-border-$color-badge-disabled-background i-bg-$color-badge-disabled-background i-text-$color-badge-disabled-text',
+  sponsored: 'i-border-$color-badge-price-background i-bg-$color-badge-price-background i-text-$color-badge-price-text',
+  neutral: 'i-border-$color-badge-neutral-background i-bg-$color-badge-neutral-background i-text-$color-badge-neutral-text',
+  roundedTopRightBottomLeft: 'rounded-tr-0 rounded-bl-0',
+  roundedTopLeftBottomRight: 'rounded-tl-0 rounded-br-0',
```
Comment (AnnaRybkina) [inline index.js:37]: I've seen usages of these classes together with ribbon so I guess it is good those are added here 👍  but I'm just curious if I missed a conversation about us deciding to add those here? Do we want our consumers to clean up a little bit or what is the purpose? 🙇‍♀️

File: index.js (hunk 1 above)
Comment (CarolineGard) [reply index.js:36]: yes, since there were no prop for it, it wasn't used. plus not tokens for it either

File: index.js (hunk 2 above)
Comment (CarolineGard) [reply index.js:37]: no there was no discussion on adding them. I simply just moved them as they were used in `Tag`
Screenshot 2023-06-09 at 09 12 58

Comment (AnnaRybkina) [reply index.js:37]: Alright, didn't now about `Tag`. Are we adding it to the docs?

File: index.js (hunk 1 above)
Comment (AnnaRybkina) [reply index.js:36]: The `ribbon` is only(for what I could see) is used as a class and not a component with props. Could not find any usages of `wTag` in finn. But I guess if we do not have token let's remove it and add if needed

File: index.js (hunk 2 above)
Comment (AnnaRybkina) [reply index.js:37]: Couldn't find any usages of it in finn at least

**Review by AnnaRybkina — APPROVED** [review]

**Pull request merged**